src/components/Display.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display(props) {
+  const { result } = props;
+  return (
+    <div className="calc-display">
+      {' '}
+      <h3>
+        Result:
+        {result}
+      </h3>
+      {' '}
```

Comment:
:thinking: I think it is cleaner to remove the `{' '}` in this code. :bulb:  You could rely on CSS if you're interested in adjusting the spacing around the `<h3>`.

Reply:
yes, thanks!